skin/frontend/base/default/js/TIG/MyParcel2014/myparcel.js

```diff
@@ -1002,19 +920,28 @@ MyParcel = {
         });
     },
 
+    /*
+    * Check if it is Saturday today, then use the Saturday CutoffTime
+    */
+    saturdayCutoffTime: function() {
```

Comment:
saturdayCutoffTime met je prefixen met
set, get, show, hide of zoiets. Ik denk nu set.

Reply:
Done